Rlabkey/man/labkey.provenance.createProvenanceParams.Rd

```diff
@@ -67,6 +70,19 @@ mi <- data.frame(lsid=c("urn:lsid:labkey.com:Sample.251.MySamples:sample1",
 p <- labkey.provenance.createProvenanceParams(name="step1", description="initial step",
         objectInputs=oi[["LSID"]], materialInputs=mi)
 
+## add run step properties and custom properties to the provenance params
+props <- data.frame(
+    "urn:lsid:labkey.com:Vocabulary.Folder-996:ProvenanceDomain#version"=c(22.3),
+    "urn:lsid:labkey.com:Vocabulary.Folder-996:ProvenanceDomain#instrumentName"=c("NAb reader"),
+    check.names=FALSE)
+params <- list()
+params$comments <- "adding additional step properties"
+params$activityDate <- "2022-3-21"
+params$startTime <- "2022-3-21 12:35:00"
+params$endTime <- "2022-3-22 02:15:30"
+params$recordCount <- 2
+p <- labkey.provenance.createProvenanceParams(recordingId=ra$recordingId, name="step2", properties=props, params=params)
```

Comment:
looks like one more CRAN NOTE to address: 

* checking Rd line widths ... NOTE
Rd file 'labkey.provenance.createProvenanceParams.Rd':
  \examples lines wider than 100 characters:
     p <- labkey.provenance.createProvenanceParams(recordingId=ra$recordingId, name="step2", properties=props, params=params)

These lines will be truncated in the PDF manual.

Reply:
sounds good, updated!